embedded-tests/src/test/resources/log4j2.xml

```diff
@@ -46,5 +46,8 @@
     <Logger level="info" name="org.apache.druid" additivity="false">
       <AppenderRef ref="RoutingAppender"/>
     </Logger>
+    <Logger level="error" name="org.apache.druid.metadata.SqlSegmentsMetadataManager" additivity="false">
```

Comment:
Yeah, this log becomes very verbose in tests since we keep a very small metadata polling period. 😅 
Suggestion: Another way to avoid these verbose logs is to set the property `druid.manager.segments.useIncrementalCache=ifSynced` on the Coordinator.